Rock/Reporting/DataSelect/Person/RelatedPeopleSelect.cs

```diff
@@ -0,0 +1,468 @@
+                                                                                                                .OrderBy( gm => gm.GroupRole.Order )
+                                                                                                                .ThenBy( gm => gm.Person.BirthDate )
+                                                                                                                .ThenBy( gm => gm.Person.Gender )
+                                                                                                                .Select(
+                                                                                                                        gm =>
+                                                                                                                        new RelatedPersonInfo
+                                                                                                                        {
+                                                                                                                            RelatedToPersonId = p.Id,
+                                                                                                                            PersonId = gm.Person.Id,
+                                                                                                                            FirstName = gm.Person.FirstName,
+                                                                                                                            LastName = gm.Person.LastName,
+                                                                                                                            Suffix = gm.Person.SuffixValue.Value,
+                                                                                                                            RelationshipName =
+                                                                                                                                ( relationshipName == "*"
+                                                                                                                                      ? gm.GroupRole.Name
+                                                                                                                                      : relationshipName )
+                                                                                                                        } ) );
+
+            return relatedPeopleQuery;
+        }
+
+        private const string _CtlFormat = "ddlFormat";
+        private const string _CtlKnownRelationshipType = "cblKnownRelationshipType";
+        private const string _CtlFamilyRelationshipType = "cblFamilyRelationshipType";
+
+        public const string FamilyRelationshipParentGuid = "A498E9DB-4BFF-4C52-A6C0-7510EECDF6E7";
```

Comment:
@MrUpsideDown Are these existing GUID's in the Rock database or new GUID's that you're creating for the report?